src/QMCHamiltonians/CoulombPBCAA.cpp

```diff
@@ -36,11 +36,12 @@ CoulombPBCAA::CoulombPBCAA(ParticleSet& ref, bool active,
   DistanceTableData *d_aa = DistanceTable::add(ref,DT_SOA_PREFERRED);
   PtclRefName=d_aa->Name;
   initBreakup(ref);
-//  if(!is_active)
-//  {
+  ref.turnOnPerParticleSK(); 
```

Comment:
PerParticleSK is not cheap.
Can it be protected by the ComputeForces flag?

Reply:
It can!  I'll do that :)